plugins/base/src/test/resources/locationProvider/android-package-list

```diff
@@ -0,0 +1,228 @@
+$dokka.format:java-layout-html
```

Comment:
Thanks for pushing a test! 
I think we could remove some potential danger here. 
Having this list here might be a little bit dangerous, because we might want to maintain it, or are confused why it is not maintained. 

What about having just the subset of this file that is actually necessary for your test being writting out by your test before running? It would also reduce the test scope!

Reply:
Yeah, but then it isn't the `android-package-list` anymore 🤔 
I just thought that it may be a good idea to have an entire file as someone might want to use it someday to their tests and then will be surprised when this file isn't complete
Anyway, left only the necessary lines 👍

Reply:
Yes, thats the whole point! 
I would warn you against offering this to others as resource, because then you should maintain it (which is uncomfortable). It might just be easer to materialize this file during the test (can be from constant string value e.g.)

Reply:
I think this won't change much so maintaining it should not be a problem but the idea of materializing those files during the tests is great. Changed to a similar thing 👍

Reply:
Awesome! Your solution is even much better 😍